playbooks/roles/mfe_deployer/defaults/main.yml

```diff
@@ -17,6 +17,12 @@ MFES:
   - name: account
     repo: frontend-app-account
     public_path: "/account/"
+  - name: payment
```

Comment:
This items must be conditional! But I do not see a proper way to make them conditional...